proms-resources/frontend/src/main/frontend/webpack.config.js

```diff
@@ -17,7 +17,9 @@ module.exports = {
     [module_name + 'Clinics']: './src/proms/Clinics.jsx',
     [module_name + 'PrintHeader']: './src/proms/PrintHeader.jsx',
     [module_name + 'WelcomeMessageConfiguration']: './src/proms/WelcomeMessageConfiguration.jsx',
-    [module_name + 'WelcomeMessageConfigurationIcon']: '@mui/icons-material/ChromeReaderMode.js'
+    [module_name + 'WelcomeMessageConfigurationIcon']: '@mui/icons-material/ChromeReaderMode.js',
+    [module_name + 'ToUConfiguration']: './src/proms/ToUConfiguration.jsx',
+    [module_name + 'ToUConfigurationIcon']: '@mui/icons-material/HelpCenter.js'
```

Comment:
How about we use the `Handshake` icon instead? Terms of Use are basically an agreement between the provider and the user @sashaandjic 

![image](https://user-images.githubusercontent.com/651980/176507566-6de8385a-1000-4985-aa82-e03becbe9b6a.png)


```suggestion
    [module_name + 'ToUConfigurationIcon']: '@mui/icons-material/Handshake.js'
```